### ISSUE TITLE
When you have an invalid post-to-beacon/beacon opp create form, it incorrectly checks all of the selected subcategories

### ISSUE THREAD

**Issue by bsmithgall** [codeforamerica/pittsburgh-purchasing-suite#285]

**Comment by ajb** [comment]
Is this something that would affect `beacon-standalone`? I can't quite ascertain from the title.

**Comment by bsmithgall** [comment]
Yes. If you go through and select categories but have, say, an invalid
email address on form submission, the validation will clear the selected
categories and you'll have to re-submit them.

This is a fairly low-priority problem based on the number of signups we get.

**Comment by ajb** [comment]
Word, thanks for filling me in.

**Comment by ajb** [comment]
This issue was ~~moved~~ copied to dobtco/beacon#3
